src/main/java/org/opensearch/knn/training/TrainingJob.java

```diff
@@ -145,12 +179,13 @@ public void run() {
             modelMetadata.setState(ModelState.FAILED);
             logger.error("Exception \"" + modelId + "\": " + e);
```

Comment:
do we need to prefix with Exception? This is error message.

Reply:
We dont even need this log given we already log it already in this function. Ill fix this.